src/components/landing/LearnMoreCard.tsx

```diff
@@ -13,7 +13,7 @@ export default function () {
       <PrivacyDeath />
       <div className={bigWhaleLabsText}>
         <BodyText>
-          <LinkText color="text-primary" url={`https://bigwhalelabs.com`}>
+          <LinkText blank color="text-primary" url={`https://bigwhalelabs.com`}>
```

Comment:
I think we should pass `url="https://bigwhalelabs.com"`, not `url={`https://bigwhalelabs.com`}`